### DISCUSSION TITLE
【灌水】一起在新艾利都看看风景把！

### DISCUSSION THREAD

**Discussion by Kk-k11** [share121/inter-knot#782] [其他]
![20240706191411](https://github.com/user-attachments/assets/510f5b15-0735-426a-8986-e4f2e360683b)
没人觉得光映广场这里咖啡店的景色很美吗？有没有人感兴趣线下来喝一杯下午茶？

**Comment by GanlinSama** [comment]
光映广场必去打卡点，在这里看夕阳真的很放松

**Reply by Kk-k11** [reply]
是吧是吧！

**Comment by aaa00073** [comment]
漫步在海边时，微微浮动的海浪能够让心灵宁静下来，阳光也很温暖。不过为什么非机动车道上既看不见飞机也看不见动车呢？

**Comment by chenger2024** [comment]
![20240711193021](https://github.com/user-attachments/assets/a12edec0-2017-4c2d-9b22-777edf19b5ba)
这里也很好看

**Comment by Kk-k11** [comment]
很宁静的地方，令人舒心

**Comment by warning-Kirov-reporting** [comment]
把新艾利都烧成灰！
